modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -256,6 +256,8 @@ if (CONFIG_BUILD_WITH_TFM)
   )
 
   zephyr_link_libraries(tfm_api)
+  zephyr_sources(
+    ${ZEPHYR_BASE}/modules/trusted-firmware-m/src/zephyr_tfm_ns_interface.c)
```

Comment:
Why is this a `zephyr_sources` ?
Does it really belong in the `zephyr` lib.

Reply:
It is part of the nonsecure image, to help interfacing with tfm.